app/tests/database.py

```diff
@@ -0,0 +1,139 @@
+from stac_api.models import Asset
+from stac_api.models import Collection
+from stac_api.models import CollectionLink
+from stac_api.models import Item
+from stac_api.models import ItemLink
+from stac_api.models import Keyword
+from stac_api.models import Provider
+from stac_api.models import get_default_stac_extensions
+
+
+def create_collection():
+    '''Create a dummy collection db object for testing
+    '''
+    collection = Collection.objects.create(
+        crs=["http://www.opengis.net/def/crs/OGC/1.3/CRS84"],
+        created='2020-10-28T13:05:10.473602Z',
+        updated='2020-10-28T13:05:10.473602Z',
+        description='This is a description',
+        start_date=None,
+        end_date=None,
+        extent=[200000, 100000, 200001, 100005],
```

Comment:
This is not compatible with the most recent collection model.
No separate start_ and end_date anymore, extent is now a JSONField covering temporal and spatial extent.